### PR TITLE
Improve Build Editor docs

### PR DESCRIPTION
**Pull request by tsmithv11**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

**Comment by k-laughman** [comment]
These are good additions @tsmithv11 = thank you very much for writing a fix for some of the explanations. I believe these slight changes will help explain to customers some of the syntax and limitations. I think these are very useful enhancements to the document.

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policy-examples.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ To create code-based policies for your infrastructure, use these examples as gui
 [#bicep-example]
 === Bicep Example
 
-The following example demonstrates how to create a Bicep policy.
+The following example demonstrates how to create a Bicep policy. This policy ensures that auditing is enabled for Microsoft SQL servers and their databases. It filters resources to include only Microsoft.Sql/servers and Microsoft.Sql/servers/databases, then checks for a connection with auditing settings for both resource types and that the settings are enabled.
```
Comment (JBakstPaloAlto) [inline docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policy-examples.adoc:15]: The following example shows how to create a Bicep policy that ensures auditing is enabled for Microsoft SQL servers and their databases. It filters resources to include only Microsoft.Sql/servers and Microsoft.Sql/servers/databases, then verifies that auditing settings are configured and enabled for both resource types.

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policy-examples.adoc [hunk 2]
```diff
@@ -71,15 +71,14 @@ If you are creating policies for Terraform, here are some examples you can use t
 [#t-basic-policy-definition]
 ==== Basic Policy Definition
 
-Use the sample guideline to create a basic policy definition using one attribute.
+In the follow example, the attribute is “tags.env”, where tags is the attribute and “env” is the key name you are searching for in a key:value pair of tags. If instead, you would like to ensure that every resource has a tag of Department:<something>, change the attribute to “tags.Department”. Note that this is case sensitive. Use the operator `equals_ignore_case` if you prefer a case insensitive check.
```
Comment (JBakstPaloAlto) [inline docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policy-examples.adoc:74]: In the following example, the attribute is `tags.env`, where `tags` is the attribute, and `env` is the key name you are searching for in a key:value pair of tags. To ensure that every resource has a `Department:<something>` tag, modify and set the attribute to `tags. Department`. Note that this is case-sensitive. Use the `equals_ignore_case` operator for a case-insensitive check.

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc [hunk 3]
```diff
@@ -39,7 +39,7 @@ The *IaC Policy Code Editor* is displayed by default in the *Create query* step
 [#configure-iac]
 === Configure IaC Custom Build Policies
 
-Limitation: The Visual Editor supports policy creation for Prisma Cloud's supported cloud service providers. For unsupported providers, use the Code Editor.
+Limitations: The Visual Editor supports policy creation for cloud service providers supported by Prisma Cloud. For unsupported providers, use the Code Editor. Additionally, the Resource Type and Attribute fields are selection only and not free text. For more advanced policies that require custom attributes, leverage the Code Editor. For example, tagging rules often require attributes like `tags.<tag_key>` which cannot be accomplished with the Visual Editor. Finally, the Visual Editor does not support connected resources, use the Code Editor for these policies.
```
Comment (JBakstPaloAlto) [inline docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc:42]: **Limitations**
-  The Visual Editor supports policy creation for cloud service providers supported by Prisma Cloud.  For unsupported providers, use the Code Editor
- The 'Resource Type' and 'Attribute' fields are selection-based, not free text
-  For more advanced policies requiring custom attributes, such as tagging rules with `tags.<tag_key>`, use the Code Editor
- The Visual Editor does not support connected resources; use the Code Editor for these policies

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc [hunk 4]
```diff
@@ -72,6 +72,23 @@ If your custom IaC code contains no errors, Prisma Cloud will display up to 30 r
 The following screenshot displays an example of results returned by a verified IaC policy.
 +
 image::governance/custom-build-policy-iac.png[]
++
+The same policy could be defined using the Code Editor using the following:
```
Comment (JBakstPaloAlto) [inline docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc:76]: The same policy could be defined using the Code Editor as follows:

**Pull request merged**